src/contact.rs

```diff
@@ -724,13 +719,17 @@ impl Contact {
                 )
                 .await?;
 
-            let self_name = context
-                .get_config(Config::Displayname)
-                .await?
-                .unwrap_or_default();
-            let self_name2 = stock_str::self_msg(context);
-
             if let Some(query) = query {
+                let self_addr = context
+                    .get_config(Config::ConfiguredAddr)
+                    .await?
+                    .unwrap_or_default();
```

Comment:
why are you not using `get_primary_self_addr().await.unwrap_or_default()` here?

Reply:
Makes sense, probably forgotten.